pulp_smash/pulp_smash_internals.ipynb

```diff
@@ -0,0 +1,989 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pulp-Smash Internals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why?\n",
+    "\n",
+    "Get familiar with Pulp-Smash, and get to know how Pulp is tested. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Jupyter Notebook\n",
+    "\n",
+    "You can also follow along using a **Jupyter Notebook**.  Assuming you are bringing your own Pulp machine. \n",
```

Comment:
Ha! Isn't this a little ``chicken or the egg`` problem?

https://en.wikipedia.org/wiki/Chicken_or_the_egg

I had to guess/figure out how to install Junyper and open a notebook only to see this as the second heading! :D

Reply:
Github renders the notebook. Relying on this to start the work flow.